packages/nx-extensions/src/executors/built-script/executor.ts

```diff
@@ -7,6 +7,11 @@ const dirname = __dirname;
 
 export default async function runExecutor(options: BuiltScriptExecutorSchema) {
 	const args = [
+		...(options['inspect'] ? ['--inspect-brk'] : []),
```

Comment:
For posterity – let's find a way to reuse `nx-extensions` between `playground-tools` and `playground` repos.

Reply:
Opened https://github.com/WordPress/wordpress-playground/issues/650 to track this

Reply:
@adamziel What do you think of this: We can make it its own NPM package, and the use a relative symlink to take the directory in `node_modules/` and put it wherever we need in the folder structure.

Git & GitHub both handle relative symlinks pretty well, and if I remember correctly windows-based git clients will translate it to/from native "shortcuts" transparently.

Reply:
I'd like to avoid a workflow where testing changes between repos requires pushing something to GitHub – would that do it? If not, then I wonder if there's a solution where we could just make a change in `nx-extensions` in a local copy of the `wordpress-playground` repo and have it automatically reflected in the local copy of the `playground-tools` repo. CC @dmsnell who spent a ton of time exploring a similar problem in context of Gutenberg and Tumblr.

Reply:
@adamziel Nope, we'd use `npm link` for that.

https://docs.npmjs.com/cli/v9/commands/npm-link

With that, you can take any package and symlink it INTO the `node_modules/` directory of another project on the same system.

The symlink that lives in `packages/nx-extensions/` won't care if it points at a directory or another symlink.

Reply:
That makes a lot of sense, thank you @seanmorris! Let's proceed with this PR as it is and then use #650 to track code reuse further down the road